apypie/action.py

```diff
@@ -94,3 +94,34 @@ def filter_empty_params(self, params=None):
                 raise InvalidArgumentTypesError
         else:
             return {}
+
+    def prepare_params(self, input_dict):
+        """
+        Transform a dict with data into one that can be accepted as params for calling the action.
+
+        This will remove any keys that are not accepted as params when calling the action.
```

Comment:
Remove implies it modifies the original dict.
```suggestion
        This will ignore any keys that are not accepted as params when calling the action.
```